app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Api::V1::CommentsController < ApplicationController
+  skip_before_action :authenticate_user!
+  protect_from_forgery unless: -> { request.format.json? }
+  def index
+    @post = Post.find(params[:post_id])
+    @comments = Comment.where(post: @post)
+    render json: @comments
+  end
+
+  def show
+    @post = Post.find(params[:post_id])
+    @comments = Comment.where(post: @post)
+    render json: Comment.find(params[:id])
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new(comment_params)
+    @comment.user = @user
+    @comment.post = @post
```

Comment:
- You have done well so far, but according to the project requirements, you are to create an API endpoint to add a comment to a post. Remember that the owner of the comment is the user who makes it; it should not be set by passing a parameter. From what you have done you are setting the user creating the comments. Please fix this 🙏 
```ruby
     @post = Post.find_by(user_id:params[:user_id], post_id:params[:post_id])
     new_comment = current_user.comments.new(comment_params)
     new_comment.post = @post
```